content/Kotlin/kotlin.md

```diff
@@ -0,0 +1,18 @@
+---
+Title: 'Kotlin'
+Description: 'Kotlin is a contemporary language released by JetBrains in 2011. Kotlin is becoming popular and is fast replacing Java as people’s choice for software projects.'
```

Comment:
```suggestion
Description: 'Kotlin is a contemporary language released by JetBrains in 2011. It is becoming popular and quickly replacing Java as the choice for Android software projects.'
```

Reply:
I think it'd also be good to match the introductory paragraph to the suggestion above